include/roaring/portability.h

```diff
@@ -72,6 +68,34 @@
 
 #endif // DISABLE_X64
 
+#ifdef _MSC_VER
+/* Microsoft C/C++-compatible compiler */
+#include <intrin.h>
+
+/* wrappers for Visual Studio built-ins that look like gcc built-ins */
```

Comment:
There are some faster versions of the built-ins used here which will function or be undefined depending on the processor features. Future work would be to identify the features and use the fancier built-ins